Makefile.am

```diff
@@ -1376,7 +1376,7 @@ libsss_idmap_la_LIBADD = \
     $(NULL)
 libsss_idmap_la_LDFLAGS = \
     -Wl,--version-script,$(srcdir)/src/lib/idmap/sss_idmap.exports \
-    -version-info 5:1:5
+    -version-info 6:0:6
```

Comment:
How does this versioning work - why not 6:0:0?

Reply:
> How does this versioning work - why not 6:0:0?

Hi,

it's libtool magic, see https://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html for details. Since I didn't remove anything I followed until step 5.

HTH

bye,
Sumit